handlers/mesh_ops_handlers.go

```diff
@@ -309,22 +299,20 @@ func (h *Handler) AdapterPingHandler(w http.ResponseWriter, req *http.Request, p
 		}
 	}
 	if aID < 0 {
-		err := errors.New("unable to find a valid adapter for the given adapter URL")
-		logrus.Error(err)
-		http.Error(w, "Adapter could not be pinged.", http.StatusBadRequest)
+		h.log.Error(ErrValidAdapter)
+		http.Error(w, ErrValidAdapter.Error(), http.StatusBadRequest)
 		return
 	}
 
 	if prefObj.K8SConfig == nil || !prefObj.K8SConfig.InClusterConfig && (prefObj.K8SConfig.Config == nil || len(prefObj.K8SConfig.Config) == 0) {
-		logrus.Error("No valid kubernetes config found.")
-		http.Error(w, `No valid kubernetes config found.`, http.StatusBadRequest)
+		h.log.Error(ErrInvalidK8SConfig)
+		http.Error(w, ErrInvalidK8SConfig.Error(), http.StatusBadRequest)
 		return
 	}
 
 	mClient, err := meshes.CreateClient(req.Context(), prefObj.K8SConfig.Config, prefObj.K8SConfig.ContextName, meshAdapters[aID].Location)
 	if err != nil {
-		logrus.Errorf("Error creating a mesh client: %v.", err)
-		http.Error(w, "Adapter could not be pinged.", http.StatusBadRequest)
+		http.Error(w, ErrMeshClient.Error(), http.StatusBadRequest)
```

Comment:
Log the Error?

Reply:
Ah sorry I forgot to do that I am doing it .